### DISCUSSION TITLE
Defining a Rule for Updating a Specific Column and Preventing Changes to Other Columns (Update Rules)

### DISCUSSION THREAD

**Discussion by Amirshp75** [pocketbase/pocketbase#5932] [Q&A] [answered]
Hello. I want to implement update rules where **the user can only update specific fields (columns) and prevent any changes to other fields**.

_**For example,**_ table fields: "Name," , "Phone Number" , "Status."
 I want the user to be able to update only the "Name" field, and any attempt to change the "Phone Number" or "Status" should be prevented.

I read the documentation, but couldn't find anything specific on this.

@ganigeorgiev 🌷🙏

**Comment by ganigeorgiev** [comment, marked as answer]
_Please don't ping me for no reason. I can't provide dedicated support._

In any case, to prevent a specific field change you can use the `:isset` modifier, for example:

```
@request.data.phone:isset = false &&
@request.data.status:isset = false
```
(_in v0.23.0 `@request.data.*` is renamed to `@request.body.*`_).

v0.23.0 docs - https://pocketbase.io/docs/api-rules-and-filters/#isset-modifier
v0.22.x docs - https://pocketbase.io/old/docs/api-rules-and-filters/#isset-modifier

**Reply by Amirshp75** [reply]
Thank you, it's solved. _I pinged because I wanted the response from the creator (i.e., you)_, otherwise, there was no specific reason. **I just wanted a quick response.** I appreciate that you respond immediately. 🌷🙏

**Reply by ganigeorgiev** [reply]
Next time when you ping me I'll not bother replying.
